src/ctypes_helpers.mli

```diff
@@ -9,6 +9,28 @@ val safe_deref : 'a Ctypes.ptr -> 'a
 
 type ulong = Unsigned.ULong.t
 
+module Reachable_ptr : sig
+  (** Pointers with a GC link.
+      These pointers are reachable through a "root"
```

Comment:
Is "root" the right word?  I'd say "parent" because it can be reachable from other objects.